package SpecialEquipConfig;

message SpecialEquipItem
{
	required uint32 eqid		= 1;  //定制的装备id
	repeated uint32 born		= 2;   //统和勇
	optional uint32 star		= 3;   //星级
	repeated uint32 subs		= 4;   //洗练属性类型
	repeated uint32 sublevel	= 5;   //洗练等级
}

message SpecialEquips
{
	repeated SpecialEquipItem  specialequip	= 1;  //装备
}